docs/source/developer_guides/custom_models.mdx

```diff
@@ -14,7 +14,7 @@ specific language governing permissions and limitations under the License.
 
 Some fine-tuning techniques, such as prompt tuning, are specific to language models. That means in 🤗 PEFT, it is
 assumed a 🤗 Transformers model is being used. However, other fine-tuning techniques - like
-[LoRA](./conceptual_guides/lora) - are not restricted to specific model types.
+[LoRA](../conceptual_guides/lora) - are not restricted to specific model types.
```

Comment:
otherwise, it is going to `https://huggingface.co/docs/peft/developer_guides/conceptual_guides/lora`

it should be `https://huggingface.co/docs/peft/conceptual_guides/lora`